frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,19 @@
+import styles from './styles.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  className?: string;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-  <button type={type}>{label}</button>
+const Button: React.FC<Properties> = ({
+  type = 'button',
+  label,
+  className = '',
+}) => (
+  <button type={type} className={`${styles.button} ${styles[className]}`}>
```

Comment:
1. There is `getValidClassNames` helper currently in development for conditional styling classes
2. Your approach won't work because when `className` is brought from upper component to button - button styles file know nothing about that class (but you're trying to get class styling exactly from button styles file